### PR TITLE
Improve scrolloff behavior

### PR DESCRIPTION
**Pull request by senekor**

### CODE REVIEW

**Comment by senekor** [comment]
alternative to https://github.com/helix-editor/helix/pull/10795

**Review by pascalkuthe — DISMISSED** [review]
lgtm if CI passes

File: helix-view/src/view.rs [hunk 1]
```diff
@@ -241,13 +241,23 @@ impl View {
...
+                scrolloff.min(viewport.height.saturating_sub(1) as usize / 2),
+                scrolloff.min(viewport.height as usize / 2),
+            )
+        };
+        let (scrolloff_left, scrolloff_right) = if CENTERING {
+            (0, 0)
         } else {
-            scrolloff.min(viewport.height.saturating_sub(1) as usize / 2)
+            (
+                // - 1 from the top so we have at least one gap in the middle.
```
Comment (the-mikedavis) [inline helix-view/src/view.rs:257]:
```suggestion
                // - 1 from the left so we have at least one gap in the middle.
```

**Review by the-mikedavis — COMMENTED** [review]
just a minor nit about a comment, otherwise this looks good

File: helix-view/src/view.rs (hunk 1 above)
Comment (senekor) [reply helix-view/src/view.rs]: good catch!

**Review by the-mikedavis — APPROVED** [review]

**Review by pascalkuthe — APPROVED** [review]

**Pull request merged**